test/include/seqan3/test/performance/seqan2_minimiser.h

```diff
@@ -0,0 +1,221 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Mitra Darvish <mitra.darvish AT fu-berlin.de>
+ * \brief Provides a SeqAn2 minimiser hash.
+ */
+ 
+#pragma once
+
+#include <seqan3/core/platform.hpp>
+
+#ifdef SEQAN3_HAS_SEQAN2
+#include <seqan/seq_io.h>
+
+//!\brief Strong type for passing the window size.
+struct window { uint64_t v; };
+//!\brief Strong type for passing the kmer size.
+struct kmer { uint64_t v; };
+//!\brief Strong type for passing number of bins.
+struct bins { uint64_t v; };
+//!\brief Strong type for passing number of bits.
+struct bits { uint64_t v; };
+//!\brief Strong type for passing number of hash functions.
+struct hashes { uint64_t v; };
+
+//!\brief Whether to use xor for computing the hash value.
+enum use_xor : bool
+{
+    //!\brief Do not use xor.
+    no,
+    //!\brief Use xor.
+    yes
```

Comment:
```suggestion
    no, //!< Do not use xor.
    yes //!< Use xor.
```
Those are equivalent. Just to show you a more compact version :)